src/wallet/wallet.cpp

```diff
@@ -649,15 +649,6 @@ bool CWallet::ChangeWalletPassphrase(const SecureString& strOldWalletPassphrase,
     return false;
 }
 
-void CWallet::chainStateFlushed(ChainstateRole role, const CBlockLocator& loc)
```

Comment:
Two calls to `chainStateFlushed` are left, in `MigrateLegacyToDescriptor` and `bench/wallet_migration.cpp`

Reply:
Removed